translation/dest/contact/en-US.xml

```diff
@@ -52,6 +52,7 @@
   <string name="knightMate">It is possible to checkmate with only a knight or a bishop, if the opponent has more than a king on the board.</string>
   <string name="noRatingPoints">No rating points were awarded</string>
   <string name="ratedGame">Make sure you played a rated game. Casual games do not affect the players\' ratings.</string>
+  <string name="botRatingAbuse">In certain circumstances when playing against a bot account, a rated game may not award points if it is determined that the player is abusing the bot for rating points.</string>
```

Comment:
Translations should be made on the lichess project on crowdin.com, otherwise they will be saved.

Reply:
Should I remove this line then? I'm not familiar with how the translations on crowdin work.

Reply:
It doesn't really matter much since it will be overwritten on the next crowdin pull anyway. But just something to keep in mind for the future.